core/include/config.h

```diff
@@ -38,6 +38,7 @@
      0x37, 0x9c, 0xba, 0xfe, 0xb9, 0x47, 0xf0, 0x1b, 0x79, 0xcc, 0x22, 0x35, \
      0x73, 0x9a, 0x8f, 0x3b, 0xb3, 0x42, 0x37, 0x84, 0x69, 0x2e}
 
+  #define QR_PUBKEY "049dff0de94387941288cd672dd94b2c78046a42c60e4e0c7d7aa967eb32f7edef43021badba3ef585842b721aff9d7ee882c2bfc3a0345536b3a322694db0b803"
```

Comment:
I'd suggest to declare this as a `const uint8_t[65]` and let the compiler do the work for you (bounds checking, ...). That way we can also remove the complexity in `qrsignatures.c` for loading this value. Something like this:

```
// echo 049dff0de94387941288cd672dd94b2c78046a42c60e4e0c7d7aa967eb32f7edef43021badba3ef585842b721aff9d7ee882c2bfc3a0345536b3a322694db0b803 \
//  | xxd -r -p | xxd -i
const uint8_t[65] = {0x04, 0x9d, 0xff, 0x0d, 0xe9, 0x43, 0x87, 0x94, 0x12, 0x88, 0xcd, 0x67,
  0x2d, 0xd9, 0x4b, 0x2c, 0x78, 0x04, 0x6a, 0x42, 0xc6, 0x0e, 0x4e, 0x0c,
  0x7d, 0x7a, 0xa9, 0x67, 0xeb, 0x32, 0xf7, 0xed, 0xef, 0x43, 0x02, 0x1b,
  0xad, 0xba, 0x3e, 0xf5, 0x85, 0x84, 0x2b, 0x72, 0x1a, 0xff, 0x9d, 0x7e,
  0xe8, 0x82, 0xc2, 0xbf, 0xc3, 0xa0, 0x34, 0x55, 0x36, 0xb3, 0xa3, 0x22,
  0x69, 0x4d, 0xb0, 0xb8, 0x03};
```

Reply:
cool. I think that helps me remove the hex decode, even better :)